docs/anna-new-york.md

```diff
@@ -0,0 +1,4 @@
+# New York
+
+- Big Apple
+- Famous for pizza and Broadway shows
```

Comment:
@annazelikman - yeah it's good to see familiar "faces", little late here in London, but manageable.